weave/trace_server/validation.py

```diff
@@ -11,11 +11,17 @@ def project_id_validator(s: str) -> str:
 
 
 def call_id_validator(s: str) -> str:
-    return validation_util.require_uuid(s)
+    try:
+        return validation_util.require_otel_span_id(s)
+    except:
```

Comment:
Need more explicit `except` here.  What are you catching?

Reply:
Logically this is similar to `require_otel_span_id(s) or require_uuid(s)`. The exceptions raised in the `require` methods cause execution to halt so this just checks if it is an OTEL UUID before allowing the `require_uuid` to throw.

I agree its a bit messy, I guess I could modify the original `require_uuid` to also check for otel format IDs if that's better, or return either a string or error from the requires and throw if it's an instance of error

Reply:
If you want to be super general then you can do `except Exception` which will catch most things.  Bare except is dangerous because it will also catch stuff like KeyboardInterrupt which you probably dont want